assets/src/components/cd/pipelines/job/PipelineJobLogs.tsx

```diff
@@ -66,6 +83,45 @@ export default function PipelineJobLogs() {
     <ScrollablePage
       heading="Logs"
       scrollable={false}
+      headingContent={
+        jobGateState === 'RUNNING' || jobGateState === 'PENDING' ? (
```

Comment:
shouldn't we be able to get the `RUNNING`, `PENDING` strings from the generated types so they aren't magic values in the code?

Reply:
yes, totally. I'll fix that.